definitions/procedures/pulp/remove.rb

```diff
@@ -163,12 +170,34 @@ def drop_migrations
     end
     # rubocop:enable Metrics/BlockLength
 
-    def delete_pulp_data(rm_cmds)
-      with_spinner('Deleting pulp2 data directories') do |spinner|
-        rm_cmds.each do |cmd|
-          execute!(cmd)
+    def delete_pulp_data
+      non_mountpoints = data_dir_removal_cmds(deletable_pulp_dirs)
+      mountpoints = pulp_data_dirs_mountpoints
+      with_spinner('') do |spinner|
+        if non_mountpoints.any?
+          spinner.update('Deleting pulp2 data directories')
+          non_mountpoints.each do |cmd|
+            execute!(cmd)
+          end
         end
-        spinner.update('Done deleting pulp2 data directories')
+        if mountpoints.any?
+          msg_for_del_mountpoints(mountpoints)
+        else
+          spinner.update('Done deleting pulp2 data directories')
```

Comment:
So the spinner is never updated if there are mountpoints?

I must admit I am not too well versed with the spinner stuff, but as a user I would have thought I'd get something like:
* am am deleting all your old stuff
* ok, I am done deleting your stuff
* (optional) BTW, I am done, but couldn't get *everything* done, here is some homework: …

Reply:
Below is what I felt should be the states, however if you think we should handle it in any better way then happy to do the changes,

* am am deleting all your old stuff -> we should show this only when we are actually deleting something?
* ok, I am done deleting your stuff -> we should show this only when we can delete all the stuff but not in partial state?
* (optional) BTW, I am done, but couldn't get everything done, here is some homework: -> for this there are explicit messages.

Reply:
Shouldn't it be `elsif non_mountpoints.any?` instead of `else` here?

Reply:
@ekohl and @evgeni improved the messaging in recent commit, let me know your thoughts.